packages/react/src/components/Tabs/tabPanel.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useRef } from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { TabPanel } from './';
+
+test('should render tab panel with two paragraphs correctly', async () => {
+  const TabPanelwithRef = () => {
+    const ref = useRef(null);
+    return (
+      <div ref={ref}>
+        <TabPanel>
+          <p>a simple paragraph</p>
+          <p>a complicated paragraph</p>
+        </TabPanel>
+      </div>
+    );
+  };
+
+  render(<TabPanelwithRef />);
+  await waitFor(() => {
```

Comment:
Just curious, what was the `waitFor` added for? This seems to work without it.

Reply:
Thank you for pointing that out. I forgot to remove it, and it is not necessary to include.